tests/kit/model_zoo/transformers/llama.py

```diff
@@ -47,10 +49,10 @@ def data_gen_for_casual_lm():
     loss_fn_for_seq_classification = lambda output: output.logits.mean()
 
     config = LlamaConfig(
-        num_hidden_layers=4,
-        hidden_size=128,
+        num_hidden_layers=8,
```

Comment:
Why use such a large model?

Reply:
it was a mistake due to automatic merge, fixed